packages/playwright-test/src/isomorphic/teleReceiver.ts

```diff
@@ -526,7 +526,7 @@ export class TeleTestCase implements reporterTypes.TestCase {
 
 export type TeleTestResult = reporterTypes.TestResult & {
   stepMap: Map<string, reporterTypes.TestStep>;
-  statusEx: reporterTypes.TestResult['status'] | 'scheduled' | 'running';
+  statusEx: reporterTypes.TestResult['status'] | 'scheduled' | 'running' | 'modified';
```

Comment:
Telereceiver is used in a few places around; we're not comfortable adding a new execution status for such a niche usecase.

Niche features should be implemented in a well-isolated manner. An easy heuristic that could be used is "could this feature be removed by only deleting lines of code"?